router/pkg/otel/attributes.go

```diff
@@ -36,7 +36,10 @@ const (
 	WgValidationCacheHit               = attribute.Key("wg.engine.validation_cache_hit")
 	WgVariablesValidationSkipped       = attribute.Key("wg.engine.variables_validation_skipped")
 	WgQueryDepth                       = attribute.Key("wg.operation.query_depth")
-	WgQueryDepthCacheHit               = attribute.Key("wg.operation.query_depth_cache_hit")
+	WgQueryTotalFields                 = attribute.Key("wg.operation.total_fields")
```

Comment:
@StarpTech is this fine?

Reply:
I think we should scope it further e.g. `wg.operation.complexity.total_fields` because it is not a common attribute and only set when complexity limits are enabled.